core/src/main/java/tc/oc/pgm/filters/modifier/PlayerBlockQueryModifier.java

```diff
@@ -0,0 +1,26 @@
+package tc.oc.pgm.filters.modifier;
+
+import javax.annotation.Nullable;
+import tc.oc.pgm.api.filter.Filter;
+import tc.oc.pgm.api.filter.query.PlayerQuery;
+import tc.oc.pgm.api.filter.query.Query;
+
+public class PlayerBlockQueryModifier extends QueryModifier<PlayerQuery> {
+
+  public PlayerBlockQueryModifier(Filter filter) {
+    super(filter);
+  }
+
+  @Nullable
+  @Override
+  protected Query modifyQuery(PlayerQuery query) {
+    if (query.getPlayer() == null) return null;
```

Comment:
```suggestion
    // Abstain when no player can be found, eg: they disconnected
    if (query.getPlayer() == null) return null;
```
I'd add a clarifying comment for future mantainers